packages/react-scripts/fixtures/kitchensink/integration/env.test.js

```diff
@@ -6,7 +6,15 @@ describe('Integration', () => {
     it('file env variables', async () => {
       const doc = await initDOM('file-env-variables')
 
-      expect(doc.getElementById('feature-file-env-variables').textContent).to.equal('fromtheenvfile.')
```

Comment:
The `('fromtheenvfile.')`, note the period at the end. And put there here: https://github.com/facebookincubator/create-react-app/pull/1344/files#diff-5bf0cb512e071c1717dd16f597def9caL4